action.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Flank'
+description: 'Run Flank from Github actions!'
+inputs:
+  version: 
+    description: 'Flank version to run'
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - name: Download flankScripts
+      run: |
+        curl -L https://github.com/Flank/flank/releases/download/flank-scripts-1.7.3/flank-scripts.jar --output flank-scripts.jar
```

Comment:
Do we want to have `1.7.3` hardcoded?

Reply:
a version of Flank scripts will be hardcoded to use the tested and stable version for Github action.
We will update it manually

Reply:
Makes sense 👍  Thanks

Reply:
But should we use `1.8.0` then?

Reply:
```suggestion
        curl -L https://github.com/Flank/flank/releases/download/flank-scripts-1.8.0/flank-scripts.jar --output flank-scripts.jar
```